cura_app.py

```diff
@@ -118,7 +118,10 @@ def get_cura_dir_path():
 # otherwise it doesn't work. This is a workaround on Linux that before we call QDesktopServices.openUrl(), we
 # switch to a directory where the user has the ownership.
 if Platform.isLinux() and hasattr(sys, "frozen"):
-    os.chdir(os.path.expanduser("~"))
+    if "OWD" in os.environ and os.path.isdir(os.environ["OWD"]):
```

Comment:
Since `OWD` should always be present when running via AppImage, I expect this will re-introduce https://github.com/Ultimaker/Cura/issues/6194 for everyone.

Reply:
If the current directory is not writeable by the user running Cura then that could cause a problem if being able to create a file in the current directory is needed to avoid #6194. Testing whether it is possible to create a file in the current directory is obviously possible but is it worth the hassle? If you want to close this PR unused, I don't care as I only put it together as someone was banging on about not being able to open files with relative path names from the command line.

Reply:
Yes, I agree, it's not really worth the hassle there.

Sorry, I'll close this PR then. It's more important that people can click links rather than being able to provide relative paths in the command line arguments.